### ISSUE TITLE
bad nightly build for stratosphere-0.5-hadoop2

### ISSUE THREAD

**Issue by MikhailErofeev** [stratosphere/stratosphere#533]
http://stratosphere-bin.s3-website-us-east-1.amazonaws.com/
Tonights build of stratosphere-0.5-hadoop2  was just 122 bytes of binaries.  Is it Travis bug, or something else?

**Comment by rmetzger** [comment]
Hey,
thanks for reporting the issue!

According to the `tools/deploy_to_maven.sh` script

```
# Build Responsibilities
# 1. Deploy to sonatype (old hadoop)
# 2. Nothing
# 3. Deploy to s3 (old hadoop)
# 4. deploy to sonatype (yarn hadoop) (this build will also generate specific poms for yarn hadoop)
# 5. Nothing
# 6. deploy to s3 (yarn hadoop)
```

It was machine 6 (https://travis-ci.org/stratosphere/stratosphere/jobs/20121364) that was responsible to deploy the file.
It seems that https://s3.amazonaws.com/archive.travis-ci.org/jobs/20121364/log.txt copying the file did not work

```
cp: cannot stat `stratosphere-dist/target/stratosphere-dist-0.5-hadoop2-SNAPSHOT-bin/stratosphere-0.5-hadoop2-SNAPSHOT/*': No such file or directory
```

I'll push a fix in a minute.

**Comment by rmetzger** [comment]
Thanks again. I pushed a fix in f8a81732afaa3d48ace99fc3f5ed01b5ef334a2d.
